input/jira/FHIR-fhircast.xml

```diff
@@ -2,13 +2,13 @@
 <specification 
                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
                ballotUrl="http://hl7.org/fhir/uv/fhircast-docs/2022May" 
-               ciUrl="https://build.fhir.org/ig/HL7/fhircast-docs" 
```

Comment:
My understanding is that this file is only generated for copying over to https://github.com/HL7/JIRA-Spec-Artifacts/ and that it otherwise has no functional impact. 

Do you disagree?